python/paddle/v2/framework/tests/test_fill_constant_batch_size_like_op.py

```diff
@@ -3,13 +3,27 @@
 from op_test import OpTest
 
 
-class TestFillConstantBatchSizeLikeOp(OpTest):
+class TestFillConstantBatchSizeLikeOp1(OpTest):
     def setUp(self):
         self.op_type = "fill_constant_batch_size_like"
         self.inputs = {'Input': np.random.random((219, 232)).astype("float32")}
-        self.attrs = {'value': 3.5, 'shape': [-1, 132, 777]}
+        self.attrs = {'value': 3.5, 'shape': [-1, 132, 7]}
 
-        out = np.random.random((219, 132, 777)).astype("float32")
+        out = np.random.random((219, 132, 7)).astype("float32")
+        out.fill(3.5)
+        self.outputs = {'Out': out}
+
+    def test_check_output(self):
+        self.check_output()
+
+
+class TestFillConstantBatchSizeLikeOp2(OpTest):
+    def setUp(self):
+        self.op_type = "fill_constant_batch_size_like"
+        self.inputs = {'Input': np.random.random((219, 232)).astype("float32")}
+        self.attrs = {'value': 3.5, 'shape': [132, -1, 7], 'dim_idx': 1}
+
+        out = np.random.random((132, 232, 7)).astype("float32")
```

Comment:
Would this RNG cause random passing/failure of unit tests?

Reply:
random seed is fixed in the base class, i.e., OpTest.